pkg/remoteconfig/avalanche.go

```diff
@@ -13,35 +13,38 @@ import (
 )
 
 type AvalancheConfigInputs struct {
-	HTTPHost         string
-	APIAdminEnabled  bool
-	IndexEnabled     bool
-	NetworkID        string
-	DBDir            string
-	LogDir           string
-	PublicIP         string
-	StateSyncEnabled bool
-	PruningEnabled   bool
-	Aliases          []string
-	BlockChainID     string
-	TrackSubnets     string
-	BootstrapIDs     string
-	BootstrapIPs     string
-	GenesisPath      string
+	HTTPHost                   string
+	APIAdminEnabled            bool
+	IndexEnabled               bool
+	NetworkID                  string
+	DBDir                      string
+	LogDir                     string
+	PublicIP                   string
+	StateSyncEnabled           bool
+	PruningEnabled             bool
+	Aliases                    []string
+	BlockChainID               string
+	TrackSubnets               string
+	BootstrapIDs               string
+	BootstrapIPs               string
+	GenesisPath                string
+	UpgradePath                string
+	ProposerVMUseCurrentHeight bool
```

Comment:
no need to use it as a variable, can be just hardcoded in the config  as it's always `true`

Reply:
lets keep the option open for the moment